### DISCUSSION TITLE
airflow 2.4.0 warning about base_log_folder

### DISCUSSION THREAD

**Discussion by alexvkaam** [apache/airflow#26826] [General]
### Apache Airflow version

2.4.0

### What happened

Hello

I always try to keep airflow up to date and after each update work out all the warning etc.

so upgrade 2.3.4 results during db upgrade

airflow db upgrade
DB: postgresql+psycopg2://airflow:***@localhost:5432/airflow
Performing upgrade with database postgresql+psycopg2://airflow:***@localhost:5432/airflow
[2022-09-27 08:43:35,472] {db.py:1466} INFO - Creating tables
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.

which is fine.

now upgrade 2.4.0 results during db upgrade

airflow db upgrade
DB: postgresql+psycopg2://airflow:***@localhost:5432/airflow
Performing upgrade with database postgresql+psycopg2://airflow:***@localhost:5432/airflow
[2022-09-27 08:08:21,693] {migration.py:204} INFO - Context impl PostgresqlImpl.
[2022-09-27 08:08:21,693] {migration.py:211} INFO - Will assume transactional DDL.
[2022-09-27 08:08:21,705] {db.py:1530} INFO - Creating tables
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
WARNI [airflow.configuration] section/key [core/base_log_folder] not found in config
WARNI [airflow.configuration] section/key [core/base_log_folder] not found in config

the interesting part is that base_log_folder has been moved to [logging] a while ago and correctly is not in [core], so I think this warning is faulty.

### What you think should happen instead

no warning

### How to reproduce

upgrade 2.3.4 to 2.4.0

### Operating System

Ubuntu 20.04.4 LTS (Focal Fossa)

### Versions of Apache Airflow Providers

apache-airflow-providers-common-sql==1.2.0
apache-airflow-providers-ftp==3.1.0
apache-airflow-providers-http==4.0.0
apache-airflow-providers-imap==3.0.0
apache-airflow-providers-postgres==5.2.1
apache-airflow-providers-sqlite==3.2.1

### Deployment

Other

### Deployment details

standalone linux server,  upgrade via 

pip3.7 install apache-airflow[postgres,password]==2.4.0 --constraint  https://raw.githubusercontent.com/apache/airflow/constraints-2.4.0/constraints-3.7.txt

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Likely some of your DAGs gets imported and retrieves log_base_folder from core section. Please review your dags and fix it.

**Reply by potiuk** [reply]
Let us know what you find please.

**Reply by alexvkaam** [reply]
good morning,

thank you, I just do the machines, not the dags, and I did not think it would import the dags when doing a airflow db upgrade with everything stopped.

But you where correct, airflow-log-cleanup.py f rom  https://github.com/teamclairvoyant/airflow-maintenance-dags/blob/master/log-cleanup/airflow-log-cleanup.py was the cause.

Thank you.

**Reply by potiuk** [reply]
BTW. cleaning logs with Airflow dags is not really good idea IMHO - DAGS should be used when you are writing some business logic with it. Cleaning logs seems like a better job for simple cronjob and https://github.com/apache/airflow/blob/main/scripts/docker/clean-logs.sh
